hilll2-HW03.py

```diff
@@ -0,0 +1,123 @@
+#TODO Do not edit this file directly! Instead, create a new file called
+#        hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Langston Hill
+# Username: hilll2
+#
+# Assignment: Homework 3: Sea of Stars
+# Purpose: To learn the ins and outs of git, along with implementing RGB values into code
+# Google Doc Link: https://docs.google.com/document/d/11EDVFiCCLZRo3eQhymX9W1aXn25xKEMBGwPpwzYjYQI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: https://colorspire.com/rgb-color-wheel/
+# https://docs.python.org/3/library/turtle.html
+#
+#
+#################################################################################
+
+
+import turtle
+import random
+def draw_star(john):
+    """
+       Draws a star of red, blue, purple, a randomly generated color, or white.
+       They are also of Different sizes
+     """
+
+    color = 0, 0, 0
+    john.pensize(random.randrange(1,4))
+    john.pendown()
+    colorgen = random.randrange(1, 10)
+    if colorgen == 1:
+        color = 255, 0, 31
+    elif colorgen == 2:
+        color = 78, 238, 255
+    elif colorgen == 3:
+        color = 193, 84, 255
+    elif colorgen == 4:
+        color = random.randrange(1,255), random.randrange(1,255),random.randrange(1,255)
+    else:
+        color = 255, 255, 255
+
+    john.pencolor(color)
+    x = random.randrange(2, 7)
+    for i in range(4):
+
+        john.forward(x)
+        john.backward(x)
+        john.left(90)
+
+def draw_sun(john):
+    """
+       Draws the sun and its rays of random lengths and widths
+    """
+    john.pensize(10)
+    john.pencolor("orange")
+    john.fillcolor("yellow")
+    john.goto(0,100)
+    john.pendown()
+    john.begin_fill()
+    john.circle(100)
+    john.end_fill()
+    john.left(90)
+    john.penup()
+    john.forward(100)
+    john.right(50)
+
+    for i in range(72):
+        x = random.randrange(75,250)
+        john.pensize(random.randrange(1,10))
+        john.penup()
+        john.forward(100)
+        john.pendown()
+        john.forward(x)
+        john.backward(x)
+        john.penup()
+        john.backward(100)
+        john.left(5)
+        pass
+    # ...
```

Comment:
Remove all template helper code (this was used to prevent a syntax error when the functions were empty). Multiple of these throughout the file.